django_tables2/tables.py

```diff
@@ -620,3 +620,39 @@ class Table(TableBase):
     __doc__ = TableBase.__doc__
 
 # Table = DeclarativeColumnsMetaclass(str('Table'), (TableBase, ), {})
+
+
+def table_factory(model, table=Table, **kwargs):
```

Comment:
I'd say, if `**kwargs` doesn't allow passing arbitrary argument to the metaclass, it provides not much benefit to not explicitly define the arguments this function expects. 

So either allow pass-through of arguments in some way, or be explicit about what arguments the function takes.